apps/builddao/widget/Feed.jsx

```diff
@@ -100,53 +100,64 @@ const MainContent = styled.div`
 
 const [currentFeed, setCurrentFeed] = useState("updates");
 
-const feed = () => {
-  if (currentFeed === "updates")
-    return (
-      <Feed
-        index={[
-          {
-            action: "post",
-            key: "main",
-            options: {
-              limit: 10,
-              order: "desc",
-              accountId: props.accounts,
-            },
-            cacheOptions: {
-              ignoreCache: true,
-            },
+const CustomFeed = (name) => {
+  name = !!name ? name : 'main'
+  return (
+    <Feed
+      index={[
+        {
+          action: "post",
+          key: name,
+          options: {
+            limit: 10,
+            order: "desc",
+            accountId: props.accounts,
           },
-          {
-            action: "repost",
-            key: "main",
-            options: {
-              limit: 10,
-              order: "desc",
-              accountId: props.accounts,
-            },
-            cacheOptions: {
-              ignoreCache: true,
-            },
+          cacheOptions: {
+            ignoreCache: true,
           },
-        ]}
-        Item={(p) => (
-          <StyledPost>
-            <Widget
-              loading={<div className="w-100" style={{ height: "200px" }} />}
-              src="/*__@appAccount__*//widget/components.post.post"
-              props={{
-                accountId: p.accountId,
-                blockHeight: p.blockHeight,
-                noBorder: true,
-              }}
-            />
-          </StyledPost>
-        )}
-      />
-    );
-  else if (currentFeed === "bugs") return "Bugs Feed";
-  else if (currentFeed === "resources") return "Resource Feed";
+        },
+        {
+          action: "repost",
+          key: name,
+          options: {
+            limit: 10,
+            order: "desc",
+            accountId: props.accounts,
+          },
+          cacheOptions: {
+            ignoreCache: true,
+          },
+        },
+      ]}
+      Item={(p) => (
+        <StyledPost>
+          <Widget
+            loading={<div className="w-100" style={{ height: "200px" }} />}
+            src="/*__@appAccount__*//widget/components.post.post"
+            props={{
+              accountId: p.accountId,
+              blockHeight: p.blockHeight,
+              noBorder: true,
+            }}
+          />
+        </StyledPost>
+      )}
+    />
+  );
+}
+
+const nameToKey = (name) => ({
+  "updates": "update",
```

Comment:
I think we could expand this to take an object, where this object has a preloaded template